assets/dev/scss/frontend/_global.scss

```diff
@@ -111,7 +111,7 @@
 		position: absolute;
 		width: 100%;
 		height: 100%;
-		z-index: 0;
+		z-index: $under-layer;
```

Comment:
This seems dangerous.
Please note this affects all background slideshows, including section and column.
Why is this necessary?